tarorpc/taro.proto

```diff
@@ -554,3 +559,13 @@ message SendAssetResponse {
 
     int64 total_fee_sats = 5;
 }
+
+message GetInfoRequest {
+}
+
+message GetInfoResponse {
+    string version = 1;
+    string lndversion = 2;
+    string network = 3;
+    int64 assets = 4;
```

Comment:
I'm not sure we should be adding this here. The `GetInfo` call should be as lightweight as possible and not do a huge DB lookup that basically fetches all assets from disk. Maybe we can add this back once we have a more efficient way to count the assets.